pubapi/v1/dto/continuous_screenings.go

```diff
@@ -0,0 +1,80 @@
+package dto
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/checkmarble/marble-backend/models"
+	"github.com/checkmarble/marble-backend/pure_utils"
+	"github.com/google/uuid"
+)
+
+type ContinuousScreeningRequest struct {
+	SearchInput json.RawMessage `json:"search_input"`
+}
+
+type ContinuousScreeningMatch struct {
+	Id                    uuid.UUID       `json:"id"`
+	ContinuousScreeningId uuid.UUID       `json:"continuous_screening_id"`
+	OpenSanctionEntityId  string          `json:"opensanction_entity_id"` //nolint:tagliatelle
+	Status                string          `json:"status"`
+	Payload               json.RawMessage `json:"payload"`
+	ReviewedBy            *uuid.UUID      `json:"reviewed_by"`
+	CreatedAt             time.Time       `json:"created_at"`
```

Comment:
You should use `pubapi.DateTime` for client-facing timestamp, which will take take of supporting different parsing format, of canonical serializing, and to produce proper error messages.